src/api/app/helpers/webui/project_helper.rb

```diff
@@ -122,4 +122,9 @@ def escape_list(list)
         "']".html_safe
     end.join(',').html_safe
   end
+
+  def show_package_actions?
+    !@project.defines_remote_instance? && !(@is_incident_project && @packages.present? &&
```

Comment:
This is a lot of conditions... I think it might be simplified a bit with some guard clauses which are splitting this up and returning early. It will be easier to follow then.